package.json

```diff
@@ -53,10 +53,12 @@
     "lite-server": "^2.2.0",
     "mocha": "3.2.0",
     "mocha-webpack": "0.7.0",
+    "moment": "2.0.0",
```

Comment:
I don't like 3rdParty dependencies but if this is working I'll accept this now and mark it as to be refactored.
Funny side fact that also moment introduces new complexity in the back^^